src/ceph-daemon

```diff
@@ -53,6 +55,88 @@ except ImportError:
 
 podman_path = None
 
+##################################
+# Popen wrappers, lifted from ceph-volume
+
+def call(command, desc, verbose=False, **kw):
+    """
+    Wrap subprocess.Popen to 
+
+    - log stdout/stderr to a logger, 
+    - decode utf-8
+    - cleanly return out, err, returncode
+
+    If verbose=True, log at info (instead of debug) level.
```

Comment:
this doesn't seem to match the implementation - seems like it would help debugging to be able to have a mode similar to 'bash -x' where all commands being run + their output is shown

Reply:
yep, fixed verbose=True case (only used by the ceph-volume command).

for general debugging, pass -d to ceph-daemon so that the logging level is debug instead of info by default